scheduler/src/cook/config.clj

```diff
@@ -530,3 +533,7 @@
   "Used to get the fremework-id"
   []
   (get-in config [:settings :mesos-framework-id]))
+
+(defn max-over-quota-jobs
+  []
+  (get-in config [:settings :max-over-quota-jobs]))
```

Comment:
This can return `nil`, doesn't look like `nil ` is handled in the uses. Should we worry about that?

Reply:
Done